tests/main/interfaces-snap-interfaces-requests-control/task.yaml

```diff
@@ -103,12 +103,9 @@ execute: |
     echo "Check snap can access rules via /v2/interfaces/requests/rules"
     api-client --socket /run/snapd-snap.socket "/v2/interfaces/requests/rules" | jq '."status-code"' | MATCH '^'"$EXPECTED_HTTP_CODE"'$'
 
-    # XXX: creating rules requires polkit authentication, so for now, use curl
-    # XXX: should use `snap debug api` instead of curl, but should really just use api-client
```

Comment:
I think I would prefer to preserve these comments, slightly modified, since even I need reminding why the `api-client` line is commented out in favor of the curl/`snap debug api` line.

```
    # XXX: creating rules requires polkit authentication, so for now, use snap debug api instead of api-client
```